hera_mc/utils.py

```diff
@@ -26,6 +26,47 @@ def str_to_bytes(s):
     def bytes_to_str(b):
         return b.decode('utf8')
 
+from astropy.time import Time
+from astropy.time import TimeDelta
+from astropy import coordinates as coord
+from astropy import units as u
+import numpy as np
+import datetime
+
+def LSTScheduler(starttime,LSTbin_size):
+    #input an astropy time object
+    #input an LSTbin_size in seconds
+    #   grid the day into regular LST bins of size LSTbin_size
+    #   return the time of the next occuring LST bin
+
+    sidesec = 365.25/366.25 #length of sidereal second in SI seconds.
+    locate=coord.EarthLocation(lon=21.25*u.deg, lat=-30.33*u.deg) #HERA location, #XXX get the HERA location programmatically
```

Comment:
yes, please get it from cm

Reply:
done, but moved that part to the script leave the function db free

Reply:
but you're still hardcoding lat.

Reply:
I wanted the function to only take the absolute minimum inputs necessary.
Longitude is really the only bit required for this approximate LST calculation. But I still needed to input latitude, so its hard-coded. Could have made it 0, but since this function is HERA specific I thought it ought to be HERA.

Reply:
ok, fair enough.